### DISCUSSION TITLE
new release

### DISCUSSION THREAD

**Discussion by kirkbig** [gonzalezreal/swift-markdown-ui#267] [General]
aloha ++

with the recent addition of `FontWidth`, could a release be made so that swift package versioning in Xcode can pick it up.  2.2.1 or 2.3, whatever you feel would be most appropriate.

(_in the meantime, i can point to `branch: main` in Xcode to pick up this change._)

thanks,
++ Kirk

**Comment by gonzalezreal** [comment]
Hey @kirkbig,

As this is a new functionality, I will publish it as part of 2.3.0. I would like to have more content to release before publishing, though. So please hold on to `main` if that works for you.
